client/public/index.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\HttpFoundation\Request;
+
+require dirname(__DIR__).'/vendor/autoload.php';
+require dirname(__DIR__) . '/app/AppKernel.php';
+
+// We don't want to load the .env files if we are running on a server - ENVIRONMENT is only injected via terraform
+if (empty($_SERVER['ENVIRONMENT'])) {
+    (new Dotenv())->load(dirname(__DIR__).'/frontend.env', dirname(__DIR__).'/admin.env');
```

Comment:
Curious why the need for this? Is it a symfony-ism? Docker should take care of loading the env files and they should be accessible from inside the PHP runtime without having to load  in like this? Unless something is really weird about our setup.

Reply:
The main driver behind this was how we were handling the environment mode and debug profiling options via a hacky local command to create a file (see https://github.com/ministryofjustice/opg-digideps/blob/66727991d164521a97a4cbbc635fc057dc9a29b6/client/web/app_dev.php#L24). I was having trouble getting Symfony to take notice of externally loaded env vars we were defining in docker-compose and as Symfony made the Dotenv component a built-in part of version 4+ I thought it would be a good idea to try to standardise how we're handling env vars.

I'll have another play around with seeing how env vars are respected to be sure there wasn't something else that was the issue as I agree it would be nice to just have one place to inject them.

Reply:
Awesome, its working without the dotenv component so I'll drop to keep things simpler.